### ISSUE TITLE
Strange power indication 

### ISSUE THREAD

**Issue by diygoodies** [Dfinitski/N7DDC-ATU-100-mini-and-extended-boards#38]
Hi I have N7DDC 7x7 tuner and [ SW2013 mini transieveк](http://www.diygoodies.org.ua/?p=1377)and  [windom antenna](http://www.diygoodies.org.ua/?p=1506). Transiever has 14-15W of full power in all bands so I expect from tuner indication of same power value, but all that I get discribed in flowing table.

![power](https://user-images.githubusercontent.com/5357334/105627009-a4bf9600-5e3c-11eb-851e-d2395f50696f.JPG)

I tryed BAT41 diodes with windom antenna, and 1N5711 diodes with windom antenna and then 50Ohm dummy load.
As for now I can't understend why Power indication is so low, and why Power indication is so non liniar when antenna is connected?
Here is a picture of my transformer
![8594b50f-9cd1-4441-a15f-a9545eba4753](https://user-images.githubusercontent.com/5357334/105627191-e997fc80-5e3d-11eb-82d9-4cd2f3454ec6.jpg)
Whaiting for advice. Thanks!

**Comment by mike-hb** [comment]
What about reducing the secondary winding down to a few turns -- the sensivity of the power bridge is increasing.

**Comment by diygoodies** [comment]
> What about reducing the secondary winding down to a few turns -- the sensivity of the power bridge is increasing.

But what about non liniar mesurments on real antenna?

**Comment by mike-hb** [comment]
Non-linear measurement results occur when measuring with low power. Your table shows only small deviations of the SWR between your two measuring devices (columns 1 and 2) only slightly above the minimum power threshold of 5 W -- Could this be the reason? What did you expect when using different rectifiers? As described in the manual, one can find out, that there is a correction of the measured values. (V 3.2 in main.h line 42-60 correction is performed with a step function with 14 steps over the full range) This implies one special type of diodes.

Hope this helps
Michael
